app/controllers/blips_controller.rb

```diff
@@ -111,7 +111,17 @@ def blip_params(mode)
   end
 
   def blip_too_old
-    redirect_back(fallback_location: blips_path, flash: {notice: 'You cannot edit blips more than 5 minutes old'})
+    respond_to do |format|
+      format.html do
+        redirect_back(fallback_location: blips_path, flash: { notice: "You cannot edit blips more than 5 minutes old" })
+      end
+      format.json do
+        render json: {
+          success: false,
+          reason: "You cannot edit blips more than 5 minutes old",
+        }, status: 422
```

Comment:
Ah, that's how you do multiline code suggestions
```suggestion
       render_expected_error(422, "You cannot edit blips more than 5 minutes old")
```